Src/Clients/WebUI/wwwroot/js/pageSwitcher.js

```diff
@@ -0,0 +1,18 @@
+﻿'use strict'
+
+class SwitchPage {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['SwitchPage' is defined but never used.](https://app.codacy.com/manual/liannoi/team-project/pullRequest?prid=5384135)